lib/webdriver.py

```diff
@@ -273,3 +280,23 @@ def _set_account_name(self, account_monitor: AccountMonitor, response: JSON) ->
             f"Successfully logged in to {account_monitor.first_name} "
             f"{account_monitor.last_name}'s account\n"
         )  # Don't log as it contains sensitive information
+
+    def _quit_driver(self, driver: Driver) -> None:
+        driver.quit()
+        self._stop_display()
+
+    def _start_display(self) -> None:
+        try:
+            self.display = Display(visible=0, size=(1440, 1880))
```

Comment:
It seems like `visible` is supposed to be a boolean and defaults to False. I'll remove it from being specified here. https://github.com/mdmintz/sbVirtualDisplay/blob/a6c725ba4e5d9cae1481bcea5bb7c708fc281a6f/sbvirtualdisplay/display.py#L15

Reply:
Should we use False for (Xvfb)

Reply:
Actually, I will add `backend = 'xvfb'` to ensure we always use XVFB